example/scripts/lib/mutex.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+class CumulusLockError extends Error {
+  constructor(message) {
+    super(message);
+    this.name = this.constructor.name;
+  }
+}
+
+class Mutex {
```

Comment:
Super nitpicky: for files that just export a class (ignoring the error class), I think the file name should be in all capitals, so `Mutex.js`. I also might be in favor of renaming this to `DeploymentMutex` given its specificity

Reply:
Fair - in this case, the specificity feels contextual, we're not including it outside the deployment/example project scope: bd4d5f2cb